mantidproject/mantid
# Add Scroll Bars To Some Tabs In Indirect

**Description of work.**
Added QScrollArea to all bayes tabs, as well as Elwin and I(Q, t) in data analysis. (Don't be scared by lines changed I have only updated .ui files so no need for a code review just test the interfaces)

**To test:**
(I suggest increasing font size if you are using a monitor or larger sized laptop to force the scroll bars on)
Open Indirect Bayes
All the tabs here should be in scroll areas and have a scroll bar if necessary
Open Indirect Data analysis
Only the Elwin and I(Q, t) tabs here are in scroll areas, the other tabs you can pop out the mini plot and fit browser

Fixes #31426 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
